src/IO/Observer/ReductionActions.hpp

```diff
@@ -48,6 +51,40 @@ struct WriteReductionData;
 /// \endcond
 }  // namespace ThreadedActions
 
+/// Protocols related to observers
+namespace protocols {
```

Comment:
Sure! I think doing things "the right way" as we go is better than waiting for big refactors. I squashed this directly since it only moved code from one file to another.